src/components/core/Profile/Profile.js

```diff
@@ -210,6 +210,20 @@ const Profile = () => {
     },
     {
       itemId: 3,
+      content: (
+        <a
+          href="https://discord.gg/NcfJu3a9kM"
```

Comment:
Why didn't you use the link from the developer console? I'd rather see Michał Cichocki on the invitation page (due to his role in Dev Rel) instead of you. Moreover, it'd be nice to put this link in enum or constant somewhere in the project.